app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -366,44 +368,7 @@ public void onResponse(Call<WebSearch> call, Response<WebSearch> response) {
                             Log.e(TAG, response.toString());
                             if (response.body() != null ) {
                                 realm.beginTransaction();
-                                RealmList<WebSearchModel> searchResults = new RealmList<>();
```

Comment:
Yes I solved that one later. That's why currently I stopped working on this and removing common part.